WordPress/src/main/res/values/strings.xml

```diff
@@ -4005,4 +4005,20 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="blogging_prompts_onboarding_try_it_now">Try it now</string>
     <string name="blogging_prompts_onboarding_remind_me">Remind me</string>
     <string name="blogging_prompts_onboarding_card_prompt">Cast the movie of your life.</string>
+
+    <!-- Initial Screen -->
+    <string name="initial_screen">Initial Screen</string>
+    <string name="initial_screen_home">Home</string>
+    <string name="initial_screen_site_menu">Site Menu</string>
```

Comment:
Nitpick: I wonder if we can point to the below strings with `translatable=false` here as this initial screen setting directly relates to these tabs and their content. If we change tab titles in the future then changes will be reflected here as well. Wdyt?

```
<string name="my_site_dashboard_tab_title">Home</string>
<string name="my_site_menu_tab_title">Site Menu</string>
```

Reply:
> Nitpick: I wonder if we can point to the below strings with `translatable=false` here as this initial screen setting directly relates to these tabs and their content. If we change tab titles in the future then changes will be reflected here as well. Wdyt?
> 
> ```
> <string name="my_site_dashboard_tab_title">Home</string>
> <string name="my_site_menu_tab_title">Site Menu</string>
> ```

Yes 👍  Perfect idea!!! I'll make the change.

Reply:
@ashiagr - I have updated the PR to include the "string" suggestion above and set the default value to site_menu for the settings. This PR is ready for review and merge (after CI). My next PR will update the implementation of the "default" tab + experiment to use this pref. It should make things easier for the default tab logic, as we'll just need to use the initial value. 🤞

Reply:
Got it! I'll hit the merge button now.